kubernetes/loculus/templates/_siloDatabaseConfig.tpl

```diff
@@ -35,6 +35,12 @@ schema:
       name: {{ .name }}
   {{- end }}
   {{- end }}
+  {{- if .files }}
+  {{- range .files }}
+    - type: string
+      name: {{ .name }}
```

Comment:
Does this mean the fileField names must be different from all other fields passed to silo? We should probably document this

Reply:
yes indeed, the metadata field names and file field names (name to be improved) should not have any overlaps. Yes we should document it.

Reply:
I've documented it now! https://github.com/loculus-project/loculus/pull/3978/commits/cb2f35c115509d01dca1ea472615e6e13eec9139